docs/gettingstarted/quickstart.ipynb

```diff
@@ -116,13 +947,281 @@
8jjnmmLBMupr7CrSoum7dOmPSpEler1Y78BUXF2cMGTLEGD58uDFixAhj0KBBHlt0HPz1xz/+scVbGR9QVVXlce/ZA1/9+/c3Ro8ebeTn53tckSzJSElJMT788MOw5HHAtGnTvG6V0rVrV2PEiBHG6aef7nHPmgMTxHvvvTfgKyuBloiW8W779u3Gb3/7W68/nwd/denSxTjhhBOM0aNHG6eeeqpx1FFHeawWOPSD38qVK8PyXq1YscLjA50kIzEx0Rg2bJgxZswYY8iQIV5/7k866aSg3xN/9u7da5xyyilex48hQ4YYY8aMMYYNG+Z1vO7SpYuxdOnSsOVSX1/v9cOo5PpF46hRo4zTTjvNyMzM9Pp/xv/7f/8vbLkA/jDeecf8LjjM79AeRMt4F4i2LKoaBvM7X5jfoT1grPOOuV1w2uvcjqIqACAq1dfXG2+//bZx6aWXGunp6SFNeMaMGWN8+umnIf0nG21F1QNKS0uNZ555xjjjjDOMuLi4oM8XFxdnXHXVVR5XuYbLa6+95nX7Fm9fZrPZuOyyy8JydbE3K1as8PrB2NfXySefHPC9fIBwauvx7oCqqirjhRdeMM4++2y/H+r8fZlMJuP00083Zs2aZTQ2NobxXTKMsrIy46abbjLi4+MDyqVz587GE088EfY8DMO1RfyTTz7p95cIB3/Fx8cbv/vd77xerR0Oc+fONYYMGRLw92ncuHHGhg0bIpIL4A/jnXfM7wLH/A7tRbSMd/48/vjjxrhx41r8dfvtt4ecA/M735jfoT1grPOOuV3g2uPczmQYhiEAAKJYY2OjVqxYoe+++05r1qzR+vXrVVRUpPLyctXU1CglJUWZmZnKysrS0UcfrVNPPVUjRozQkUceGfI5hw4dqtWrV4fxVXi3aNEijRw5MqS21dXVWrx4sX744QetXbtWP/30k0pLS1VeXq6Ghgalp6crIyNDOTk5GjZsmE499VSdfvrp6tSpU3hfxCEaGxv12Wef6ZNPPtF//vMf/fLLL6qqqlJsbKw6d+6sQYMGadSoUbr44ot1xBFHRDQXSfruu+/0/vvva+nSpdq8ebPKy8slSenp6erfv79OOeUUXXDBBTrppJMingvQnLYY77ypra3VkiVL9OOPP2r9+vXauHGjioqKVFlZqaqqKsXExCg1NVWpqanq1q2bhg4dqmOPPVYjR45Unz59wprLofbt26d33nlHX3zxhdauXat9+/aptrZWycnJ6tGjh4499lhNmDBB559/vhITEyOaS21trT788EPNnTtXP/74o3bu3Knq6molJCQoJydHQ4YM0RlnnKGLL75YXbp0iWguhmFo0aJF+uijj/Ttt9/q559/VkVFhSwWizIzMzVgwACdfvrpuvjiizV48OCI5gIEgvHOO+Z3gWF+h/YkWsa7aMb8zjvmd2hPGOu8Y24XmPY0t6OoCgAAAAAAAAAAAAB+mNs6AQAAAAAAAAAAAACIZhRVAQAAAAAAAAAAAMAPiqoAAAAAAAAAAAAA4AdFVQAAAAAAAAAAAADwg6IqAAAAAAAAAAAAAPhBURUAAAAAAAAAAAAA/KCoCgAAAAAAAAAAAAB+UFQFAAAAAAAAAAAAAD8oqgIAAAAAAAAAAACAHxRVAQAAAAAAAAAAAMAPiqoAAAAAAAAAAAAA4AdFVQAAAAAAAAAAAADwg6IqAAAAAAAAAAAAAPhBURUAAAAAAAAAAAAA/KCoCgAAAAAAAAAAAAB+UFQFAAAAAAAAAAAAAD8oqgIAAAAAAAAAAACAHxRVAQAAAAAAAAAAAMAPiqoAAAAAAAAAAAAA4AdFVQAAAAAAAAAAAADwg6IqAAAAAAAAAAAAAPhBURUAAAAAgMPA0qVLNX/+/LZOA1GipqZGTz31lBoaGto6FQAAAOCwQFEVAAAAANoxwzA0Z84cPfHEE1q0aFFbp4M28PPPP+uiiy7Saaedpg0bNrR1OogSDQ0Nmjx5sgYOHKj33nuvrdMBAAAA2j2KqgAAAABwiGnTpslkMkXN17Rp07zm6XA4NHHiRE2cOFG33XabRo8erRtvvLF13yy0qddff11HH310U9HMZDL5PLZPnz4B/5uLjY3Vtm3bWutlqL6+Xj179gw4v1mzZnntJxw/b96EY0xITExURkaG+vTpo1NPPVWXXXaZHnzwQc2ZM0cVFRVhf08PvJaCggJddNFFuuyyy2Sz2cJ+HgAAAKCjoKgKAAAAAO3UrFmzNHfuXLfY888/ry+//LKNMkJrcTgcmjJlin71q1/JbrdLks4++2xdc801Yem/sbFRDz/8cFj6CsS//vUv7dq1q9XO1xZqa2tVXl6u7du3a9myZZo9e7buvfdeTZw4UVlZWTrttNP0zDPPqLS0NCznS0tL0z/+8Q/FxMRIkmbPnq38/PxWLZYDAAAAh5OYtk4AAAAAAKJZQkKCRowYEfDxpaWlWrlypUd82LBhyszMDLifr776SrW1tX6PWbVqlc/46NGjAz4X2hen06nLLrtM7777blPsD3/4g55++mmZzb6vnR4xYoT27dvnFvv5559VUFDg9fhXXnlF9957r3r27BmexH1oaGjQjBkzfD4/btw4j1j37t0DPlaS1q5dqz179rjFgvnZ7tevn0ffu3fv1rp16zyOPf3005WYmOgWczgcqqmpUVlZmXbs2OGxYtThcGjp0qVaunSpbr/9dl133XW6++671a1bt4Dy8+XGG29U//79dc4558hms2nNmjU66aSTtGzZMvXr169FfQMAAAAdjckwDKOtkwAAAACAaDJt2jT95S9/kST17t1bv/zyS8BtFy9erFGjRnnEFy1apJEjRwbcT58+fbR9+3ZJ0v333+91C+AXXnhBv/3tbz3i8+bN81lcQvs3efJkPfXUU02PL774Ys2ePdvv1r++HPxv3ZubbrpJzz77bChpBuzFF1/UDTfc4PP5cPza4je/+Y1eeeUVt1iwP9uHmjVrlq6++mqP+LZt29SnTx+
```

Comment:
Can we add error bars here? That would make it pretty hard to read by band, but maybe we could separate the bands to different subplots and/or limit the number of bands we show.